PassIO/app.py

```diff
@@ -60,8 +60,32 @@ def eventapproval():
     return render_template('eventapproval.html',user=CurrentUser, unapproved_events=unapproved_events)
 @app.route('/editevent')
 def editevent():
-    event = mongo.db.Event.find({"verified": "verified"})
-    return render_template('editevent.html',user=CurrentUser, event=event)
+    event_id = request.args.get('id')  # Get event ID from query parameter
```

Comment:
I think we can integrate this with what I have done in the "Event history" page for hosts, where it display all the events hosted by the specific logged in host:
<img width="1147" alt="Screenshot 2024-04-09 at 9 19 56 PM" src="https://github.com/COSC310-Team-Passio/PassIO-COSC310-EventTicketProject/assets/60246843/f03939b3-4f66-4841-a232-67051b4cb93a">
, cuz I think currently hosts can edit events not hosted by them.

Let's merge to main and ill make the changes after.